src/main/kotlin/org/rust/ide/formatter/settings/RsCodeStyleSettings.kt

```diff
@@ -15,6 +15,8 @@ class RsCodeStyleSettings(container: CodeStyleSettings) :
     @JvmField var ALIGN_WHERE_CLAUSE = false
     @JvmField var ALIGN_TYPE_PARAMS = false
     @JvmField var ALIGN_WHERE_BOUNDS = true
+    @JvmField
+    var INDENT_WHERE_CLAUSE = true
```

Comment:
Let's place annotation and variable to one line to have the same code style

Reply:
I think I was punked here by IntelliJ's "reformat code" setting in the commit dialog.